CHANGES/7686.bugfix

```diff
@@ -0,0 +1,2 @@
+Supporting url formats that conform to ansible-galaxy cli (e.g. "https://galaxy.ansible.com" and
+"https://galaxy.ansible.com/api").
```

Comment:
Do we want to allow slashes on the end here? It's kind of a silly thing, but the docs I'm reading have slashes. @daviddavis wdyt?

Reply:
https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client

Reply:
Yup, I can do it.

Reply:
[Done](https://github.com/pulp/pulp_ansible/pull/397/files#diff-d8871ad5c606deb92237ce777a88530a3522c2ae112096a0e105fb42f8337cf7R75).